src/utils/utils.go

```diff
@@ -0,0 +1,34 @@
+package utils
+
+import "strings"
+
+func ParseArgs(args []string) map[string][]string {
```

Comment:
Let's not return a map because we're actually only expecting two lists (either of which could maybe be empty)